app/main.py

```diff
@@ -1,6 +1,39 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+API_KEY = os.getenv("API_KEY")
+FILTERING = "Barcelona"
```

Comment:
![image](https://github.com/mate-academy/py-docker-weather-api/assets/104974881/4416a649-23a7-4407-b992-f20e76690d4f)
why did you write "Barcelona"?)